src/utils/language.ts

```diff
@@ -17,6 +22,11 @@ function predictLanguage(text: string): { lang: string; accuracy: number } | und
   return detectAll(cleanedText)?.[0]
 }
 
+export async function predictLanguageForArray(texts: string[]) {
+  const result = await languageDetectionInstace.detect(texts)
+  return result.map((row) => row[0].language)
+}
+
 export function predictVideoLanguage({ title, description }: any): string | undefined {
```

Comment:
Why is this function still being used in the mappings to predict language? This is the old way.

I think it's better to remove it al together